__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app.js");
+
+describe("GET /api/topics", () => {
+  test("calls this endopoint and sees if it is an array or not", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+      expect(Array.isArray(res.body)).toBe(true);
+      });
+  });
+  test('should return an endpoint(an array of objects) that has the properties slug', () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
```

Comment:
We could condense all these tests into 1. Also it would be better to also test the data you are getting back. For example what datatype are you expecting description to be?